actors/builtin/market/policy.go

```diff
@@ -40,3 +54,16 @@ func DealWeight(proposal *DealProposal) abi.DealWeight {
 	dealSpaceTime := big.Mul(dealDuration, dealSize)
 	return dealSpaceTime
 }
+
+func dealQAPower(dealSize abi.PaddedPieceSize, verified bool) abi.StoragePower {
```

Comment:
both `dealProviderCollateralBounds` and `dealQAPower` lgtm

Reply:
thanks for the quick review!